src/server/api/routers/teacher.ts

```diff
@@ -12,7 +12,7 @@ export const teacherRouter = createTRPCRouter({
     .query(async ({ ctx, input }) => {
       const teacher: Teacher | null = await ctx.db.teacher.findUnique({
         where: {
-          user_id: input.teacherId,
+          id: input.teacherId,
```

Comment:
this needs to be changed backk i think it's not working on your end because you can't log in :( since the `user_id` is connected with login